src/components/NutrientForm.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react'
+
+import InputField from '../components/InputField'
+// import SelectField from '../components/SelectField'
+import {
+  calculateProteinRate,
+  calculateFatRate,
+  calculateCarbRate,
+} from '../components/nutrientForm.helpers'
+
+import Box from '@mui/material/Box'
+import FormControl from '@mui/material/FormControl'
+import Button from '@mui/material/Button'
+import Alert from '@mui/material/Alert'
+import Stack from '@mui/material/Stack'
+
+const NutrientForm = () => {
+  const [inputs, setInputs] = useState({})
+  const [proteinRate, setProteinRate] = useState(0)
+  const [fatRate, setFatRate] = useState(0)
+  const [carbRate, setCarbRate] = useState(0)
+
+  const handleInputChange = (event) => {
+    setInputs((prevState) => ({
+      ...prevState,
+      [event.target.name]: event.target.value,
```

Comment:
This key directly set from the form name is a bit dangerous.
Can you do the enum to store and use the key too?